### PR TITLE
Fabricraft merge

### PR DESCRIPTION
**Pull request by jagpreetsinghsasan**
#46

### CODE REVIEW

File: platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml [hunk 1]
```diff
@@ -52,6 +52,8 @@ spec:
         args:
         - |-
           #!/usr/bin/env sh
+          if [ {{ $.Values.consensus.name }} == kafka ]
```
Comment (sownak) [inline platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml]: Shouldn't this have ''?

File: platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer.tpl [hunk 2]
```diff
@@ -2,11 +2,11 @@ Orderer: &OrdererDefaults
   OrdererType: {{ consensus.name }}
   Addresses:
 {% for orderer in orderers %}
-  {%- if provider == 'minikube' %}
+{% if provider == 'minikube' %}
```
Comment (sownak) [inline platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer.tpl]: removing - will keep newlines. Is this needed?

File: docs/source/operations/fabric_networkyaml.md [hunk 3]
```diff
@@ -396,7 +396,7 @@ The fields under `orderer` service are
 |-------------|----------------------------------------------------------|
 | name                        | Name of the Orderer service                                                                                                     |
 | type          | This type must be `orderer`  |
-| consensus                   | Consensus type, for example: kafka                                                                               |
+| consensus                   | Consensus type, for example: kafka, etcdraft                                                                               |
```
Comment (sownak) [inline docs/source/operations/fabric_networkyaml.md]: call it "raft" like we do in quorum

File: platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer.tpl (hunk 2 above)
Comment (jagpreetsinghsasan) [reply platforms/hyperledger-fabric/configuration/roles/create/configtx/templates/configtxOrderer.tpl]: the list looks better with new lines
earlier it was like 
addresses:
  - a.com   - b.com   -c.com

now it is 
addresses:
  - a.com
  - b.com
  - c.com

File: platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml (hunk 1 above)
Comment (jagpreetsinghsasan) [reply platforms/hyperledger-fabric/charts/orderernode/templates/deployment.yaml]: I dont think we need quotes (as it is not the first character of the string.
I have tested both kafka and raft and both seems fine with this

File: docs/source/operations/fabric_networkyaml.md (hunk 3 above)
Comment (jagpreetsinghsasan) [reply docs/source/operations/fabric_networkyaml.md]: sure. I will do the changes

**Pull request merged**